### PR TITLE
Feature/connect to api

### PR DESCRIPTION
**Pull request by mmsesay**
In this PR I worked on the following:

- Connect to the books API to post, delete and fetch data
- Save the data to the redux store
- Pass the data to the child to render
- Update the store on any actions
- Validate that the form is not empty before a submitting

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/api/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+
+export const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/'; 
+export const appId = 'bJTtiRJfwS5VIiVRUXvf';
+
+export const getBooksFromApi = async () => {
+  const finalData = [];
+  const response = await axios.get(`${baseUrl}${appId}/books`);
+  // eslint-disable-next-line array-callback-return
```
Comment (mikenath223) [inline src/redux/api/index.js]: - [x] Kindly remove this eslint disable line its best to fix the issue in the code.

**Comment by mikenath223** [comment]
- [x] Kindly ensure you use the same eslint rules as stated [here](https://github.com/microverseinc/linters-config/tree/master/react-redux). You seem to have added more rules than listed.

File: src/redux/api/index.js (hunk 1 above)
Comment (mmsesay) [reply src/redux/api/index.js]: Hello, thanks for the review. I have removed it and all other eslint disable line in the other files

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**